.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "/usr/bin/python"
```

Comment:
You'll want to put this into your `.gitignore` file so it doesn't automatically get checked in.